suite/tests/CMakeLists.txt

```diff
@@ -99,6 +99,8 @@ endif (UNIX)
 set(main_run_list
   # our main configuration
   "SHORT::-code_api"
+  # no_intercept_all_signals tests.
+  "SHORT::ONLY::signal_intercept$::-no_code_api -no_intercept_all_signals"
```

Comment:
This should be UNIX-only right?

Reply:
I assumed the `.signal_intercept` suffix check would filter out non-UNIX already. Do you recommend adding a "UNIX::" option here (it doesn't exist yet)?

Reply:
Right but it seems nice to label it explicitly as OS-specific.  There's `LIN` which looks like is actually UNIX since it doesn't look for Linux but rather not-WIN32.

Reply:
Done.